recipe/meta.yaml

```diff
@@ -28,13 +28,13 @@ requirements:
     - pip
     - cython
     - numpy
-    - libgdal
+    - gdal
     - proj
     - setuptools
   run:
     - python
     - setuptools >=0.9.8
-    - libgdal
+    - gdal
```

Comment:
Does it matter if we use `libgdal-core` or `gdal` here? The `gdal` package on conda-forge just depends on `libgdal-core` + `python` according to https://github.com/conda-forge/gdal-feedstock/blob/be22293716b72a071b62aba04871dd5ced698787/recipe/meta.yaml#L634-L636, so I guess it's ok?

Reply:
Yeah, I believe that is the way to go.  Then `mamba install -n dev python=3.12 rasterio` should work out of the box.